packages/quantic/force-app/main/default/lwc/quanticResultDate/quanticResultDate.js

```diff
@@ -8,7 +8,9 @@ import {LightningElement, api} from 'lwc';
  * The `QuanticResultDate` component displays a given result date field value.
  * @category Result Template
  * @example
- * <c-quantic-result-date result={result} label="Date" field="date"></c-quantic-result-date>
+ * <template if:true={result.raw.date}>
+ *   <c-quantic-result-date result={result} label="Date" field="date"></c-quantic-result-date>
+ * </template>
```

Comment:
It "breaks" but it's a controlled break. It displays red text saying there was a component error and logs an error in the console explaining that the given field was not found. I went with this approach which is the same as atomic because I think it offers a better developer experience. This way the component you added doesn't just disappear silently but it does require you to add a template rendering condition. I added that in the template to facilitate that. What's your opinion?

Reply:
I think the closer Atomic and Quantic are in terms of behavior, the better. So I'm ok doing it that way.

The only thing that can be annoying is that the error will come up as a surprise when one result matches the result template condition, but not this component requirements. It makes it difficult for the developer building the search page to be confident that results will always render correctly.

Reply:
Yeah that's a valid concern as well. I'm not sure how to reconcile that for now but we can pay attention to developer feedback and questions and address it then.